src/compiler/axis.ts

```diff
@@ -0,0 +1,299 @@
+import {getter, setter} from '../util';
+import {Encoding} from '../Encoding';
+import * as util from '../util';
+import {X, Y, COL, ROW} from '../consts';
+import {Q, O, N, T} from '../consts';
+
+import * as time from './time';
+
+export default function(name: string, encoding: Encoding, layout, stats) {
```

Comment:
This is as intended. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import